src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom.py

```diff
@@ -8,10 +8,12 @@
 from __future__ import print_function
 from sys import stderr
 
+from azure.storage.blob import BlockBlobService#, AppendBlobService, PageBlobService
```

Comment:
Yes. I didn't trust pylint in VSCode when it said they were unused, so I just commented them out until the tests passed and forget to remove them. :D

---
In reply to: [74279277](https://github.com/Azure/azure-cli/pull/673#discussion_r74279277) [](ancestors = 74279277,74279181)